docs/tutorials/nested_spectra.ipynb

```diff
@@ -0,0 +1,183 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using Nested-Pandas with Astronomical Spectra"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In Astronomy, a spectrum is a measurement (or combination of measurements) of an object that shows the intensity of light emitted over a range of energies. In this tutorial, we'll walk through a simple example of working with spectra from the Sloan Digital Sky Survey (SDSS), in particular showing how it can be represented as a `NestedFrame`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, we'll use `astroquery` and `astropy` to download a handful of spectra from SDSS:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from astroquery.sdss import SDSS\n",
+    "from astropy import coordinates as coords\n",
+    "import astropy.units as u\n",
+    "import nested_pandas as npd\n",
+    "\n",
+    "# Query SDSS for a set of objects with spectra\n",
+    "pos = coords.SkyCoord(\"0h8m10.63s +14d50m23.3s\", frame=\"icrs\")\n",
+    "xid = SDSS.query_region(pos, radius=3 * u.arcmin, spectro=True)\n",
+    "xid_ndf = npd.NestedFrame(xid.to_pandas())\n",
+    "xid_ndf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This initial query returns a set of objects with spectra (as specified by the `spectro=True` flag). To actually retrieve the spectra, we can do the following:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Query SDSS for the corresponding spectra\n",
+    "sp = SDSS.get_spectra(matches=xid)\n",
+    "sp"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The result is a list of FITS formatted data, from this point there are a few ways that we could move towards a nested-pandas representation. The most straightforward is to build a \"flat\" spectra table from all the objects."
```

Comment:
Perhaps a bit better explanation of flat'' spectra. I know you mean not nested, but it would be valuable to explain better what it might mean.